docs/source/user_guide/gcm_based_inference/index.rst

```diff
@@ -6,6 +6,7 @@ GCM-based inference (Experimental)
     :glob:
 
     introduction
+    draw_samples
     answering_causal_questions/index
```

Comment:
I would put this _after_ `answering_causal_questions/index`. Answering causal questions is the main objective (in my opinion) and so it should come after this.

Reply:
Changed it accordingly.